parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -30,14 +30,22 @@ pub use weights::WeightInfo;
 )]
 #[scale_info(skip_type_params(M, N))]
 #[codec(mel_bound())]
-pub struct MessageBundle<M: Get<u32>, N: Get<u32>> {
+pub struct MessageBundle<M: Get<u32>, N: Get<u32>, AccountId> {
 	source_channel_id: u8,
+	account: AccountId,
 	/// Unique nonce for to prevent replaying bundles
 	#[codec(compact)]
 	nonce: u64,
-	messages: BoundedVec<Message<M>, N>,
+	messages: BoundedVec<EnqueuedMessage<AccountId, M>, N>,
```

Comment:
Nah, I didn't intend for `EnqueuedMessage` to be used here. Should only be used for the `MessageQueue` storage item (which I see you've updated correctly), so that we can identify the owners of the messages in the queue.